docs/explanations/architecture/styles.md

```diff
@@ -498,3 +496,68 @@ Similarly to block supports, there can be only one instance of any style in use
 4. **Only blocks using block supports are shown in the Global Styles UI**
 
 The global styles UI in the site editor has a screen for per-block styles. The list of blocks is generated dynamically using the block supports from the `block.json` of blocks. If a block wants to be listed there, it needs to use the block supports mechanism.
+
+### Structural layout styles
```

Comment:
Hmm, should we call them "Structural" layout styles given that they apply equally to site-level containers and small blocks like Social Icons? (Also we could argue that Global styles are fairly structural too) We could go with just "Layout styles" I guess 😄

Reply:
Oh, good point! I've removed the word structural — I agree, it was a bit superfluous.